install_files/ansible-base/roles/noble-migration/tasks/main.yml

```diff
@@ -1,17 +1,47 @@
 ---
 
+- name: Initialize
+  set_fact:
+    # Keep in sync with upgrade.rs
+    stages:
+      - 'None'
+      - 'PendingUpdates'
+      - 'MigrationCheck'
+      - 'DisableApache2'
+      - 'Backup'
+      - 'BackupIptables'
+      - 'Marker'
+      - 'SuspendOSSEC'
+      - 'ChangeAptSources'
+      - 'AptGetUpdate'
+      - 'AptGetFullUpgrade'
+      - 'AptGetAutoremove'
+      - 'RestoreIptables'
+      - 'ReenableUnattendedUpdates'
+      - 'ReenableOSSEC'
+      - 'Reboot'
+      - 'SwitchUbuntuSources'
+      - 'IntegrityCheck'
+      - 'ReenableApache2'
+      - 'RemoveBackup'
+      - 'Done'
+
 - name: Check host's migration state
   ansible.builtin.slurp:
     src: /etc/securedrop-noble-migration-state.json
   register: migration_json
-  ignore_errors: yes
```

Comment:
This was a mistake and shouldn't have been there to start with - the slurp has to succeed.